py/surveyqa/summary.py

```diff
@@ -120,31 +137,46 @@ def get_summarytable(exposures):
     darks = np.array([counts[i] for i in np.arange(2,len(counts), 4)])
     calibs = np.array([counts[i] for i in np.arange(3,len(counts), 4)])
 
+    #finding medians
+    med_air = get_median('AIRMASS', exposures)
+    med_seeing = get_median('SEEING', exposures)
+    med_exptime = get_median('EXPTIME', exposures)
+    med_transp = get_median('TRANSP', exposures)
+    med_sky = get_median('SKY', exposures)
+
     source = ColumnDataSource(data=dict(
         nights = list(Counter(exposures['NIGHT']).keys()),
         totals = list(Counter(exposures['NIGHT']).values()),
         brights = brights,
         grays = grays,
         darks = darks,
-        calibs = calibs
+        calibs = calibs,
+        med_air = med_air,
+        med_seeing = med_seeing,
+        med_exptime = med_exptime,
+        med_transp = med_transp,
+        med_sky = med_sky,
     ))
 
     #adds links to nightly pages to the nights column
-    outdir = os.path.join(os.getcwd(), 'survey-qa')
-    os.makedirs(outdir, exist_ok=True)
     template_str = '<a href="night-<%= nights %>.html"' + ' target="_blank"><%= value%></a>'
-
+    
     columns = [
-        TableColumn(field='nights', title='NIGHT', formatter=HTMLTemplateFormatter(template=template_str)),
-        TableColumn(field='totals', title='Total Exposures'),
-        TableColumn(field='brights', title='Bright Exposures'),
-        TableColumn(field='grays', title='Gray Exposures'),
-        TableColumn(field='darks', title='Dark Exposures'),
-        TableColumn(field='calibs', title='Calibrations'),
+        TableColumn(field='nights', title='NIGHT', width=100, formatter=HTMLTemplateFormatter(template=template_str)),
+        TableColumn(field='totals', title='Total', width=50),
+        TableColumn(field='brights', title='Bright', width=50),
+        TableColumn(field='grays', title='Gray', width=50),
+        TableColumn(field='darks', title='Dark', width=50),
+        TableColumn(field='calibs', title='Calibs', width=50),
+        TableColumn(field='med_exptime', title='Median Exp. Time', width=100),
+        TableColumn(field='med_air', title='Median Airmass', width=150),
+        TableColumn(field='med_seeing', title='Median Seeing', width=150),
+        TableColumn(field='med_transp', title='Median Transparency', width=150),
+        TableColumn(field='med_sky', title='Median Sky', width=150),
```

Comment:
Do Bokeh tables have the ability to restrict the number of significant digits?  If so, please only include 2 digits beyond the decimal.  If not, pre-filter the arrays to round to only 2 digits beyond the decimal.